diff -u -r --text a/arch/genie/Program.cs b/arch/genie/Program.cs
--- a/arch/genie/Program.cs
+++ b/arch/genie/Program.cs
@@ -377,7 +377,7 @@
             if (verbose)
                 Console.WriteLine ($"genie: running command '{string.Join(' ', commandLine)}'");
 
-            var commandPrefix = new string[] {"shell", "-q", $"{realUserName}@.host", Config.GetPrefixedPath ("libexec/genie/runinwsl"),
+            var commandPrefix = new string[] {"shell", "-q", $"{realUserName}@.host", Config.GetPrefixedPath ("lib/genie/runinwsl"),
                     Environment.CurrentDirectory };
 
             var command = commandPrefix.Concat(commandLine);
